src/main.jsx

```diff
@@ -6,8 +6,6 @@ import { BrowserRouter } from "react-router-dom";
 
 ReactDOM.createRoot(document.getElementById("root")).render(
   <BrowserRouter>
-    <React.StrictMode>
-      <App />
-    </React.StrictMode>
```

Comment:
Leave the react strict mode tags around the app, outside of BrowserRouter. It helps us catch errors and weird behaviours while we're developing!